src/main/resources/liquibase/crop_changelog/15_4_1.xml

```diff
@@ -0,0 +1,190 @@
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet id="v15.4.1-1" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*) > 1 FROM cvtermprop WHERE `value`='Study Detail' AND type_id=1800
+				AND cvterm_id IN (8583, 8584, 8265, 8690, 8700, 8830, 8386, 8720, 8620, 8740, 8750, 8760, 8770,
+									8780, 8387, 8790, 8383, 8194, 8195, 8193, 8191, 8192, 8196, 8800, 8810,
+									8820, 8730, 8384, 8170, 8186);
+			</sqlCheck>
+		</preConditions>
+		<comment> REMOVE Study Detail AS DEFAULT TYPE FOR environment-LEVEL/plot-LEVEL variables. IBP-3889</comment>
+		<sql dbms="mysql" splitStatements="true">
+			SET FOREIGN_KEY_CHECKS=0;
+
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8782'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8172';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8174';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8112'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8191';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8193'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8218'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8152';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8196';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8179'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8200';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8202';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8204'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8206';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8208';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8154'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8210';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8146';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8080'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8082';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8078';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8074'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8076';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8084';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8212';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8214';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8216';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8198'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8148';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8062'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8068';
+
+		       SET FOREIGN_KEY_CHECKS=1;
+		</sql>
+	</changeSet>
+	
+	<changeSet id="v15.4.1-2" author="rowena">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM cvterm WHERE cvterm_id = 6000;
```

Comment:
Hi @rvtulod, there seems to be a disconnect here. The precondition is checking the existence of a `cvterm` record? But the alter script is inserting to `location` table?

Reply:
Changed to check existence of LOCATION record.